packages/interbit/src/tests/integrationTests/helpers.js

```diff
@@ -0,0 +1,41 @@
+const log = require('../../log')
+
+const waitForState = (chainInterface, predicate = () => {}, timeout = 2000) =>
+  new Promise((resolve, reject) => {
```

Comment:
The predicate shouldn't have a default value, or the default should return a boolean (true will always return immediately, whatever the state).

Reply:
:+1: